docs/home/contribute/writing-a-blog-post.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Writing a Blog Post
+reviewers:
+- zacharysarah
+- kbarnard10
+- sarahkconway
+---
+
+{% capture overview %}
+This page shows you how to submit a post for the [Kubernetes blog](https://kubernetes.io/blog).
+
+You’ll receive a response within 5 business days on whether your submission is approved and information about next steps, if any.
+{% endcapture %}
+
+{% capture prerequisites %}
+Create a fork of the Kubernetes documentation repository as described in
+[Creating a Documentation Pull Request](/docs/home/contribute/create-pull-request/).
+{% endcapture %}
+
+{% capture steps %}
+
+## Add a new Markdown file
+
+Add a new Markdown (`*.md`) to `/blog/_posts/`.
+
+Name the file using the following format:
+```
+YYYY-MM-DD-Title.md
+```
+For example:
+```
+2015-03-20-Welcome-to-the-Kubernetes-Blog!.md
+```
+
+## Add front matter to the file
+
+Add the following block to the top of the new file:
+```
+---
+layout: blog
+title: <title>
+date:  <date>
+---
+```
+
+For example:
+```
+---
+layout: blog
+title: Welcome to the Kubernetes Blog!
+date:  Saturday, March 20, 2015
+---
+```
+
+## Create a new pull request (PR)
+
+When you [create a new pull request](/docs/home/contribute/create-pull-request/), include the following in the PR description:
+
+- Desired publishing date
```

Comment:
> Is there any expectation on feedback/back and forth/etc for reviewing the blog posts and getting them published?

@kbarnard10 👋 How much expectation setting would you like to do here for the feedback process?

> Is it cool to say "post this as soon as you're happy with the content", or do all posts have a pre-selected publication date and reviews need to be resolved before then?

Another good question for @kbarnard10. How much more would you like to say about publishing dates?

> are there different reviewers/editors for the blog section of the site, and if so - how should a contributor communicate with them (is there an @name team grouping that can be used)?

There are, in #7610. Adding an alias would be a repo setting rather than a repo content setting. I've started [@kubernetes/kubernetes-blog-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-blog-maintainers/members). I'll add @kbarnard10 and @sarahkconway once they join the K8s org.